docs/aio.md

```diff
@@ -50,57 +58,57 @@ interface AIOPin {
-### AIO.open
-
-`AIOPin open(AIOInit init);`
+AIO API
+-------
+### aio.open(AIOInit)
+* 'AIOInit' *object*  The AIOInit object has a single field called "pin"
+  that represents the name of the pin (either an integer or a string,
+  depending on the board).
 
-The `init` object lets you set the pin number. You can either use a raw
+When setting the pin number, you can either use a raw
 number for your device or use the board support module such as
 [Arduino 101](./boards/arduino_101.md) or [K64F](./boards/frdm_k64f.md) to
 specify a named pin.
 
-Use the AIOPin object returned to read values from the pin.
-
-### AIOPin.read
+Returns an AIOPin object that be used to read values from the pin.
 
-`unsigned long read();`
+AIOPin API
+----------
+### pin.read()
 
-Returns the latest reading from the pin. Blocks until it gets the result.
+Returns the latest reading from the pin (an unsigned integer). Blocks until it gets the result.
 
-### AIOPin.readAsync
+### pin.readAsync(ReadCallback)
+* 'ReadCallback' *callback* User-provided callback function that takes
+  a single unsigned integer and has no return value.
 
-`void readAsync(ReadCallback callback);`
-
-Pass a function for `callback` that will be called later when the result is
+Pass a function for `ReadCallback` that will be called later when the result is
 obtained.
 
 *WARNING: Making an async call like this allocates some memory while the call
-is pending, so if you issue them faster than they are fulfilled, you will
+is pending; if async calls are issued faster than they are fulfilled,
+the system will
 eventually run out of memory - pretty soon on these small devices. So the best
-practice would be to make sure you only have a small, fixed number pending at
+practice would be to have only a small, fixed number pending at
 any given time.*
 
 *NOTE: This function will probably be replaced with a version that instead
 returns a promise.*
 
-### AIOPin.on
-
-`void on(string eventType, ReadCallback callback);`
+### pin.on(eventType, ReadCallback)
+* 'eventType' *string* Type of event; currently, the only supported
+  type is "change".
+* 'ReadCallback' *callback* User-provided callback function that takes
+  a single, unsigned integer and has no return value; can be null. 
 
-Currently, the only supported `eventType` is 'change', and `callback` should
-be either a function or null. When a function is passed for the change event,
-the function will be called any time the analog voltage changes. (At the moment,
+The callback function is called any time the analog voltage changes. (At the moment,
 it actually gets called periodically even when it hasn't changed.) When null is
 passed for the change event, the previously registered callback will be
 discarded and no longer called.
 
-### AIOPin.close
-
-`void close();`
+### pin.close()
 
-Closes the AIOPin. Once it is closed, all event handlers registered will no
+Closes the AIOPin. Once it is closed, all registered event handlers will no
```

Comment:
Nice English / voice edits - thanks.

Reply:
I'm as human as the next guy, so I make these errors, too.
I'm also as OCD as the next guy, so expect more of these... :-)